service/gradle/testing.gradle

```diff
@@ -212,3 +212,20 @@ task pactTests(type: Test) {
   environment.put('pact.provider.version', "$project.version")
 }
 
+
+// verify the provider side of pacts WSM has with other services
+task verifyPacts(type: Test) {
+    useJUnitPlatform {
+        includeTags "pact-verification"
+    }
+
+    // TODO: do the correct/idiomatic thing here with config values, help needed for WSM specifics, desired behavior:
+    // default: use the dsp-eng-tools version
+    // allow optional local-override for local pactbroker
```

Comment:
https://github.com/DataBiosphere/terra-workspace-data-service/pull/364 does this in the config file: https://github.com/DataBiosphere/terra-workspace-data-service/pull/364/files#diff-cc6240f713f8c21e8355ffde96d288c175673fec877cc6b1b0f0ebd0edda5e57